vividus-plugin-web-app/src/main/java/org/vividus/steps/ui/web/ScriptSteps.java

```diff
@@ -46,8 +47,14 @@ public class ScriptSteps
      *
      * @param jsFileName Value of the <i>src</i> attribute
      * @return webElement found js script
+     * @deprecated Use step:
+     * This step "Then a javascript file with the name '$jsFileName' is included in the source code" will be removed
      */
+    @Deprecated(since = "0.6.6", forRemoval = true)
     @Then("a javascript file with the name '$jsFileName' is included in the source code")
+    @Replacement(versionToRemoveStep = "0.7.0",
+            replacementFormatPattern = "Then number of elements found by `xpath(.//script[contains(@src()='%1$s'])` "
```

Comment:
we need to find all elements regardless of their visibility, check source code of the step to info on the applied visibility attributes

Reply:
the same for the step below

Reply:
not fixed